app/views/map/_maps.html.erb

```diff
@@ -6,7 +6,10 @@
     <% end %>
     <h4><a href="<%= node.path %>"><%= node.title %></a></h4>
     <p style="color:#888;"><small>
-      by <a href="/profile/<%= node.map.authorship %>"><%= node.map.authorship %></a> 
+      by 
```

Comment:
Hi! It looks like this "by" should go inside the conditional `if` block, or if there's no author, it'll just say "by" and nothing after it, right? Would you mind moving "by" to the same line as the `<a>` tag 2 lines down?